### ISSUE TITLE
Wifi board OpenWRT password is set

### ISSUE THREAD

**Issue by omiq** [UCTRONICS/Smart-Robot-Car-Arduino#7]
If I try to SSH to the wifi module it asks for a password but the password is not provided anywhere.

How can I reset the password so I can access my board?

**Comment by UCTRONICS** [comment]
@omiq 
In order to prevent users from misoperation, we have not provided the password. 
If you want to know the password, you can contact me via email. support@uctronics.com.

**Comment by UCTRONICS** [comment]
@omiq 
Yes, Our operational people will tell you the detail password and operations
